src/components/password.components.js

```diff
@@ -0,0 +1,97 @@
+import $ from 'jquery';
+
+export default class PasswordInput {
+    constructor(queryString, valid = false, disabled = false, required = true) {
+        this.div = $(queryString);
+        this.input = $(`${queryString} input[type="password"]`);
+        this.icon = $(`${queryString} input[type="password"]+i`);
+
+        this.valid = valid;
+        this.disabled = disabled;
+        this.required = required;
+
+        this.checkQuery(this.div, this.input, this.icon);
+        this.addClickEvent();
+    }
+
+    checkQuery(div, input, icon) {
+        if (div.length === 0) {
+            throw new Error('Failed to query div');
+        }
+
+        if (input.length === 0) {
+            throw new Error('Failed to query input password');
+        }
+
+        if (icon.length === 0) {
+            throw new Error('Failed to query icon');
+        }
+    }
+
+    addClickEvent() {
```

Comment:
I don't have methods for these behaviors
![image](https://user-images.githubusercontent.com/23269934/163170330-43a32769-c0ad-47f3-bbeb-8807eb6f636a.png)